Tests/PointFreeTests/EmailTests/__Snapshots__/WelcomeEmailTests/testWelcomeEmail1.1.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" style="height:100%;font-size:14px;font-family:-apple-system,Helvetica Neue,Helvetica,Arial,sans-serif;line-height:1.5;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;-ms-box-sizing:border-box;-o-box-sizing:border-box;box-sizing:border-box">
```

Comment:
the image screenshots of the welcome emails look off:

![](https://raw.githubusercontent.com/pointfreeco/pointfreeco/486ec992aeacfb2c541938d07d548c8fe7b011c8/Tests/PointFreeTests/EmailTests/__Snapshots__/WelcomeEmailTests/testWelcomeEmail1.1.png)

know what's going on?

Reply:
I think those are old? I'll double-check but I regened screen shots till the markdown was rendering OK.